util/envDynawo.sh

```diff
@@ -457,23 +565,113 @@ set_cpplint() {
   fi
 }
 
-display_environmentVariables() {
-  set -x
-  set_standardEnvironmentVariables
-  set +x
+display_environment_variables() {
+  echo ADEPT_BUILD_DIR $ADEPT_BUILD_DIR
```

Comment:
Very hard to maintain. Would be better to call all our environment DYNAWO_* and display all environment variables starting with that pattern. Might be done in another commit though.

Reply:
I agree but the previous was not satisfactory for me and not very useful. this one could be used for logs if maintained correctly. Starting all by DYNAWO is something I would have done, and it avoids settings variables that could exist otherwise either